contents/blog/the-posthog-array-1-8-0-extra.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2020-06-10
+title: Array 1.8.0 extra
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+We are working on a huge user-facing feature but felt pushing it out in a rush to make an update wouldn't be wise. We decided against releasing an update this time around for that reason, but there are still plenty of fun things to check out.
```

Comment:
I'd have done something a touch less apologetic, like "We're working on some very cool features at the moment but we're not ready to release those yet, so no update this week."

Reply:
Reword to make this better